### ISSUE TITLE
Limited functionality

### ISSUE THREAD

**Issue by mplys** [leftstick/teams-ubuntu#10]
Hi,

Thanks for your app. I've successfully installed it on my Ubuntu 16.04. But it, also as web Teams version, doesn't have the ability to do video & audio calls. When on Windows client of Teams this ability exists.

As I understand you just use your app as the wrapper for web interface?

**Comment by leftstick** [comment]
Yes, a wrapper is what i have done. Thanks for letting me know this limitation, i will investigate first
